keras_cv/models/feature_extractor/clip/clip_tokenizer.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import regex as re
+import tensorflow as tf
+import tensorflow_text as tf_text
+try:
+    from keras_nlp.tokenizers import BytePairTokenizer
+except ImportError:
+    raise ImportError(
```

Comment:
This will still leave `keras_cv` unimportable if `keras_nlp` is not installed, which is not what you want. Instead do

```python
try:
    import keras_nlp
except ImportError:
    keras_nlp = None

below in CLIPTokenizer.__init__ ...
if keras_nlp is None:
    raise ValueError("ClipTokenizer requires keras-nlp. {install instructions...}")
```

This way, we error on usage, not on import.

Reply:
done